recipes/gemmi/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "gemmi" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/project-gemmi/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: aa6eb56202c6c624b587c95bde20bc48e505fe4e4a58a8516009c62f1a9f7535
+
+build:
+  number: 0
+  skip: true  # [win and py27]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja  # [win]
+  host:
+    - python
+    - pip
+    - pybind11
+    - zlib
+  run: 
+    - python
+    - zlib
```

Comment:
zlib can be removed here

Reply:
Can you explain why? My understanding is that it is required by the resulting executable. The build output contains the following:

    INFO (gemmi,bin/gemmi): Needed DSO lib/libz.so.1 found in conda-forge::zlib-1.2.11-h14c3975_1004

Reply:
zlib should have a run_export definded. It does not hurt to add it, but it should not be needed imho. https://python-packaging-tutorial.readthedocs.io/en/latest/conda.html#run-exports

Reply:
Ah, nice!